espnet2/enh/layers/iva.py

```diff
@@ -0,0 +1,513 @@
+from typing import List, Optional, Tuple
+
+import torch
+
+
+class LaplaceModel(torch.nn.Module):
+    def __init__(self, eps: Optional[float] = None):
+        super().__init__()
+        self._eps = eps if eps is not None else eps_models["laplace"]
+
+        # just so that training works
+        self.fake = torch.nn.Parameter(torch.zeros(1))
```

Comment:
If a `torch.nn.Module` does not have any parameters, an error is thrown when starting training. Or it used to be like that a while back...

Reply:
Sorry I could not think of a case where this may happen. Could you elaborate a bit more on it?

Reply:
Hum... It seems indeed to be working :)
This has happened to me a while ago, it may have been a problem with pytorch lightning rather than torch itself. I will remove it.

Reply:
OK. Thanks!